README-PL.md

```diff
@@ -0,0 +1,144 @@
+- ["Browse good first issues to start contributing to open source"](https://github.blog/2020-01-22-browse-good-first-issues-to-start-contributing-to-open-source/) - GitHub pomaga ci znaleźć dobrą pierwszą kwestię, aby zacząć kontrybuowanie do projektu Open Source.
+- ["How to Contribute to Open Source Project" by Maryna Z](https://rubygarage.org/blog/how-contribute-to-open-source-projects) - Ten artykuł skierowany jest głównie w kierunku zespołów biznesowych (ale może okazać się użyteczny dla indywidualnych autorów) gdzie mówi o tym Czemu, Jak i po co kontrybuować do projektów Open Source
+- ["start-here-guidelines" by Andrei](https://github.com/zero-to-mastery/start-here-guidelines) -Wskazówki jak działa społeczność GitHuba i na jakich zasadach opiera się kontrybuowanie do projektów Open Source. Szczególnie zaprojektowane do celów edukacyjnych i do celów praktycznych.
+
+
+
+## Bezpośrednie kategorie wyszukiwania kwestii na GitHubie
+Linki wyszukiwania które wskazują na kwestie skierowane dla początkujących. Jest to dobre miejsce aby zacząć swoją przygodę z kontrybuowaniem.
+- [is:issue is:open label:beginner](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Abeginner)
+- [is:issue is:open label:easy](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Aeasy)
+- [is:issue is:open label:first-timers-only](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Afirst-timers-only)
+- [is:issue is:open label:good-first-bug](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Agood-first-bug)
+- [is:issue is:open label:"good first issue"](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A"good+first+issue")
+- [is:issue is:open label:starter](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astarter)
+- [is:issue is:open label:up-for-grabs](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs)
+
+## Środowisko kontrybucji Mozilli
+- [Good First Bugs](https://bugzil.la/sw:%22[good%20first%20bug]%22&limit=0) - bugi które programiści zidentyfikowali za dobre do wprowadzenia do projektu.
+- [Mentored Bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=mentor%3A%40) - bugi które mają mentora przypisanego, który będzie na IRC aby pomóc ci kiedy utkniesz podczas naprawiania owego błędu.
+- [Bugs Ahoy](https://www.joshmatthews.net/bugsahoy/) - Strona dedykowana do znajdowania błędów na Bugzilli.
+- [Firefox DevTools](http://firefox-dev.tools/) - Strona dedykowania do znajdowania błędów dla profesjonalnych narzędzi przeznaczonych na przeglądarkę Firefox.
+- [What Can I Do For Mozilla](https://whatcanidoformozilla.org/) -  Dowiedz się nad czym możęsz pracować poprzez odpowiedźna kilka pytań dotyczących twoich umiejętności i zainteresowań. 
```

Comment:
"Dowiedz się (...) i zainteresowań." -> "Dowiedz się, nad czym możesz pracować poprzez odpowiedź na kilka pytań dotyczących twoich umiejętności i zainteresowań."
A lot of mistakes.